access/src/main/java/edu/unc/lib/dl/ui/controller/AdvancedSearchFormController.java

```diff
@@ -49,8 +49,8 @@ public String handleRequest(Model model, HttpServletRequest request){
 			SearchResultResponse collectionResultResponse = queryLayer.getCollectionList(accessGroups);
 			model.addAttribute("collectionList", collectionResultResponse.getResultList());
 			
-			FacetFieldObject departmentFacetFieldObject = queryLayer.getDepartmentList(accessGroups);
-			model.addAttribute("departmentList", departmentFacetFieldObject.getValues());
+			SearchResultResponse departmentFieldObject = queryLayer.getDepartmentList(accessGroups, null);
+			model.addAttribute("departmentList", departmentFieldObject);
```

Comment:
This appears to break the advanced search page somehow -- at least on my VM. It works in master.

This is the exception I got: https://gitlab.lib.unc.edu/snippets/47